addons/udes_priorities/models/priorities.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+
+
+class UdesPriority(models.Model):
+    _name = "udes_priorities.priority"
+    _order = "sequence,id"
+
+    name = fields.Char()
+    reference = fields.Char(help="Unique identifier for the priority")
+
+    def _default_sequence_id(self):
+        return self.env.ref("udes_priorities.sequence_default")
+
+    sequence = fields.Integer(default=_default_sequence_id, help="Used to order the priorities")
+    description = fields.Text(help="A description about the perpose of the priority")
+    picking_type_ids = fields.Many2many(
+        "stock.picking.type",
+        string="Stock Transfer Types",
+        help="The stock picking types this priority is visible on. No picking types means all picking types",
+    )
+    group_ids = fields.Many2many(
+        "udes_priorities.priority_group", column1="priority_id", column2="group_id"
+    )
+
+    active = fields.Boolean(default=True)
+
+    _sql_constraints = [
+        ("priority_reference_unique", "unique(reference)", "reference must be unique")
+    ]
+
+    @api.multi
+    def _number_of_oustanding_pickings(self):
+        Picking = self.env["stock.picking"]
+        self.ensure_one()
```

Comment:
(Feel free to ignore/resolve): If we're doing `self.ensure_one()` we could just use `@api.one` instead of `@api.multi`.

Reply:
Ignore this, looks like `@api.one` isn't suitable for this method, plus it's been removed from Odoo from v13 onwards.